lib/flutter_camera_ml_vision.dart

```diff
@@ -110,7 +110,7 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>>
       _lastImage = '${tempDir.path}/${DateTime.now().millisecondsSinceEpoch}';
       try {
         await _cameraController.initialize();
-        await _cameraController.takePicture(_lastImage);
+        await _cameraController.takePicture();
```

Comment:
if `_lastImage ` is not used anymore it break the pause/stop functionality who show the last frame of the video.